wellcad/com/_borehole.py

```diff
@@ -44,8 +44,13 @@ def version_minor(self):
 
     @property
     def version_build(self):
-        """int: The build number of WellCAD."""
-        return self._dispatch.VersionBuild
+        """int: The build number of WellCAD (deprecated)."""
+        return self._dispatch.VersionPatch
```

Comment:
You can't change this here, because it will then fail for WellCAD <v5.8. Instead, just leave it as `VersionBuild`. The property is still available in v5.8, but also deprecated.

Reply:
Ok, I reverted the change for the _borehole.py file

Reply:
No, still keep the "(deprecated)" text.